mesheryctl/internal/cli/root/system/start.go

```diff
@@ -108,13 +108,15 @@ func start() error {
 	if utils.PlatformFlag == "" {
 		utils.PlatformFlag = currCtx.GetPlatform()
 	}
-	if utils.PlatformFlag == "docker" || utils.PlatformFlag == "kubernetes" {
-		currCtx.SetPlatform(utils.PlatformFlag)
-	} else {
-		if utils.CfgFile != utils.DefaultConfigPath {
+	if utils.PlatformFlag != "" {
```

Comment:
So currCtx.GetPlatform() failed and the `mesheryctl system start` command was run without the -p flag. 
This  might not be the very first instance of `mesheryctl system start` as bash (one-liner) script takes platform argument. Is the installer script run in a different way? Or
Someone intentionally emptied out the platform value from current context?